src/core/core.tooltip.js

```diff
@@ -845,7 +847,7 @@ module.exports = function(Chart) {
 			// Find Active Elements for tooltips
 			if (e.type === 'mouseout') {
 				me._active = [];
-			} else {
+			} else if (e.x > globalChartArea) {
```

Comment:
The global variable will cause problems when there are multiple charts on the same page that could potentially have different sizes. Is there a reason that `me._chart.chartArea` doesn't work?

Reply:
Actually `me._chart.chartArea` works, I wasn't sure how to call it. Thanks!